forms-web-app/Dockerfile

```diff
@@ -9,6 +9,7 @@ ENV VERSION="${VERSION}"
 ENV PORT=3000
 # Do not rely on NODE_ENV - exists for performance reasons only
 ENV NODE_ENV=production
+RUN npm run sass
```

Comment:
This might need to be captured somewhere else in the build process?

Right now it's a manual step.

Reply:
Seems to have broken the build.

Reply:
The idea with the Dockerfile is that the artifact is built before the `docker build...` is run. It's probably more useful having an `npm run build` step which consists of:
```
{
  "scripts": {
    "prebuild": "rm -Rf ./dist",
    "build": "npm run build:sass && npm run build:app",
    "build:app": "cp -Rf ./src ./dist",
    "build:sass": "<command to build sass files in ./dist>"
  }
}
```

Then change all the commands that are `src/server.js` to be `dist/server` and change the `ADD src src` to `ADD dist dist` in the Dockerfile

Reply:
OK, changes made: https://github.com/foundry4/appeal-planning-decision/pull/75/commits/9281a716ab82f670be6f56a28341ce9c02e55d68#diff-3c01ce3ff376b8c4349da777b73024a11cb3b3ead3cec6cd97ae22eae0928996R6

I have left `start:dev` using `src/..` 

Dockerfile change: https://github.com/foundry4/appeal-planning-decision/pull/75/commits/9281a716ab82f670be6f56a28341ce9c02e55d68#diff-a1db04c115605ab99fc2908fe6981ed94ef22946dffc7cc860bef97514e7e58fR12

`Add . .` would be copying in both `src` and `dist` afaik, so have been more explicit to just copy in `dist` - lmk if wrong though!